test/recipes/30-test_evp_data/evpkdf_tls12_prf.txt

```diff
@@ -43,7 +43,16 @@ Output = d06139889fffac1e3a71865f504aa5d0d2a2e89506c6f2279b670c3e1b74f531016a253
 
 # Missing digest.
 KDF = TLS1-PRF
+Ctrl.Pedantic = pedantic:0
```

Comment:
Is this needed here and in other places? It is FIPS specific..

Reply:
Current implementation performs key size check in set_ctx_params function.

The secret to set in this test case is only 8 bits.
Therefore, it is needed to set pedantic flag to zero to make it pass through the logic for checking digest.

Reply:
Make this test case only available in default provider.